uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -186,9 +186,28 @@ public static function getAvailableColors() {
    * @return array An array containing the values
    */
   public static function getValuesArray($id) {
-    $result = civicrm_api3('AbsenceType', 'get', array('id' => $id));
+    $result = civicrm_api3('AbsenceType', 'get', ['id' => $id]);
```

Comment:
Any reason why we are not using the BAO to retrieve the Absence Type details here. We can cast the object to an array after fetching. 
Just thinking it will reduce unnecessary API overhead.